client/simple.go

```diff
@@ -10,12 +10,15 @@ import (
 
 var (
 	// 503 error type informing rollup to failover to other DA location
-	ErrServiceUnavailable = fmt.Errorf("eigenda service is unavailable")
+	ErrServiceUnavailable = fmt.Errorf("eigenda service is temporarily unavailable")
 )
 
-// TODO: Add support for custom http client option
 type Config struct {
-	URL string
+	URL string // EigenDA proxy RPC URL
```

Comment:
nit: it's actually a REST-ish (not quite kek) endpoint, not RPC

Reply:
[a2e80ee](https://github.com/Layr-Labs/eigenda-proxy/pull/209/commits/a2e80ee0dd07827280a4383a2753acb8f4e4169e)